torch/csrc/jit/passes/cuda_graph_fuser.h

```diff
@@ -29,5 +31,15 @@ struct TORCH_API RegisterCudaFuseGraph
   }
 };
 
+struct CudaFuserComparisonCallback {
+  using callback_type =
+      std::function<void(const Stack&, const Stack&, const std::string&)>;
+  bool run_fallback;
```

Comment:
When would this ever be `false` ?

Reply:
@eellison if you just want to collect the graph (e.g. for logging fusion groups), without running the fallback